src/App.spec.ts

```diff
@@ -195,35 +213,34 @@ describe('App', () => {
   describe('#start', () => {
     it('should pass calls through to receiver', async () => {
       // Arrange
-      const dummyReturn = Symbol();
       const dummyParams = [Symbol(), Symbol()];
-      const fakeReceiver = createFakeReceiver(sinon.fake.resolves(dummyReturn));
+      const fakeReceiver = new FakeReceiver();
       const App = await importApp(); // tslint:disable-line:variable-name
 
       // Act
       const app = new App({ receiver: fakeReceiver, authorize: noopAuthorize });
+      sinon.spy(app, 'start');
```

Comment:
what makes this line necessary?

Reply:
It isn't, I believe this is leftover from previous expects I had in here.